tests/Feature/Connect/LeaderboardInfoTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Connect;
+
+use App\Models\Game;
+use App\Models\Leaderboard;
+use App\Models\System;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+use Tests\TestCase;
+
+class LeaderboardInfoTest extends TestCase
+{
+    use BootstrapsConnect;
+    use RefreshDatabase;
+
+    public function testLeaderboardInfo(): void
+    {
+        $now = Carbon::now();
+        Carbon::setTestNow($now);
+
+        /** @var System $system */
+        $system = System::factory()->create();
+        /** @var Game $game */
+        $game = Game::factory()->create(['ConsoleID' => $system->id]);
+        /** @var Leaderboard $leaderboard */
+        $leaderboard = Leaderboard::factory()->create(['GameID' => $game->id]);
+
+        $this->get($this->apiUrl('lbinfo', ['i' => $leaderboard->id]))
+            ->assertStatus(200)
+            ->assertExactJson([
+                'Success' => true,
+                'LeaderboardData' => [
+                    'ConsoleID' => $system->id,
+                    'ConsoleName' => $system->name,
+                    'Entries' => [],
+                    'GameID' => $game->id,
+                    'GameIcon' => $game->ImageIcon,
+                    'GameTitle' => $game->title,
+                    'ForumTopicID' => $game->ForumTopicID,
```

Comment:
rcheevos [doesn't use the game or system fields](https://github.com/RetroAchievements/rcheevos/blob/4357a1420db9f8c45353c365b6e1e126ba549144/src/rapi/rc_api_info.c#L193-L199). Maybe we could use this opportunity to remove them? I don't think anything is calling the endpoint directly.

Reply:
Makes sense. Went ahead and did this in the `lbinfo` routine rather than `GetLeaderboardData()`, as the helper function is also shared by the leaderboard info page.

`lbinfo` now has:
```php
// Remove fields that are unused by rc_client.
unset(
    $leaderboardData['GameTitle'],
    $leaderboardData['ConsoleID'],
    $leaderboardData['ConsoleName'],
    $leaderboardData['ForumTopicID'],
    $leaderboardData['GameIcon'],
);
```
Which is called if a leaderboard with `$lbID` is indeed found.

Reply:
They don't appear to be getting used there either. I commented out the code setting them in leaderboard.php and the leaderboard info page loaded without errors. It's actually getting that data directly from the model: https://github.com/RetroAchievements/RAWeb/blob/85b02885dbca9bfb484f45f14c3fb7c85eba5935/resources/views/pages-legacy/leaderboardinfo.blade.php#L41-L46

In fact, it looks like the only thing following `$lbData[` in the file is `'Entries'`.

Reply:
Got it- I naively assumed they were used there. Removed in latest and reverted the `lbinfo` changes.